tests/queries/0_stateless/03611_overlay_database_server_side.sql

```diff
@@ -0,0 +1,86 @@
+-- Tags: no-parallel
+-- { echo }
+DROP DATABASE IF EXISTS dboverlay;
```

Comment:
You can also use `CLICKHOUSE_DATABASE_1` parameter, that way the test can be marked as parallel again, i.e.

```suggestion
DROP DATABASE IF EXISTS {CLICKHOUSE_DATABASE_1:Identifier};
```

(here and below)

Reply:
@azat this is the only remaining issue is that it causes the tests to fail because each time the database has a different name is there a way to specify in reference that this is a variable ?

Reply:
Aha, I see, we do not expand query parameters for echo, good catch. Let's use your approach then (just get back the `no-parallel`)